pkgs/applications/networking/browsers/tor-browser-bundle-bin/default.nix

```diff
@@ -90,19 +90,19 @@ let
   fteLibPath = makeLibraryPath [ stdenv.cc.cc gmp ];
 
   # Upstream source
-  version = "9.0.9";
+  version = "9.5";
 
   lang = "en-US";
 
   srcs = {
     x86_64-linux = fetchurl {
       url = "https://dist.torproject.org/torbrowser/${version}/tor-browser-linux64-${version}_${lang}.tar.xz";
-      sha256 = "0ws4s0jn559j1ih60wqspxvr5wpqww29kzk0xzzbr56wfyahp4fg";
+      sha256 = "sha256-CPygaVSxEZKRsdKY9ZaD6bRL1CjbEhWjxWLzN7/4jlA=";
```

Comment:
Please don't use any SRI hashes in `nixpkgs`. We still have Nix 2.0 as minimum version which doesn't support those. See also #89275.